cddl/concise-mid-tag.cddl

```diff
@@ -69,14 +106,12 @@ end-claims=109
 id-claims=124
 
 identity-value = {
-  ? device-id => device-id-type,
-  key-material => COSE_KeySet, ; see RFC8152, draft-ietf-cose-x509
+  ? 0 => $device-id-type, ; device id
+  1 => COSE_KeySet, ; key material
 }
-device-id=125
-key-material=126
-device-id-type /= #6.47115(ueid)
-ueid = bstr .size 33
-device-id-type /= #6.37(bytes .size 16) ; see RFC4412
+
+$device-id-type /= tagged-ueid
```

Comment:
If we're using it as a device instance identifier here, does it make sense to use it as a module class identifier in module-metadata?

Reply:
👍  Will fix.